parsl/tests/test_data/test_file_staging.py

```diff
@@ -1,88 +1,25 @@
-import os
 import pytest
+
 from parsl.app.app import bash_app
 from parsl.data_provider.files import File
 
 
 @bash_app
-def cat(inputs=[], outputs=[], stdout=None, stderr=None):
-    infiles = ' '.join([i.filepath for i in inputs])
-    return """echo {i}
-    cat {i} &> {o}
-    """.format(i=infiles, o=outputs[0])
-
-
-@pytest.mark.usefixtures('setup_data')
-@pytest.mark.issue363
-def test_files():
-
-    if os.path.exists('cat_out.txt'):
-        os.remove('cat_out.txt')
-
-    fs = [File('data/' + f) for f in os.listdir('data')]
-    x = cat(inputs=fs, outputs=[File('cat_out.txt')],
-            stdout='f_app.out', stderr='f_app.err')
-    d_x = x.outputs[0]
-    print(x.result())
-    print(d_x, type(d_x))
-
-
-@bash_app
-def increment(inputs=[], outputs=[], stdout=None, stderr=None):
-    # Place double braces to avoid python complaining about missing keys for {item = $1}
-    return """
-    x=$(cat {i})
-    echo $(($x+1)) > {o}
-    """.format(i=inputs[0], o=outputs[0])
+def cat(inputs=(), outputs=(), stdout=None, stderr=None):
+    infiles = " ".join(i.filepath for i in inputs)
+    return f"cat {infiles} &> {outputs[0]}\n"
```

Comment:
no \n here - it shouldn't harm anything though

Reply:
Addressed in a follow-up PR.